modules/avro-exporter/src/main/scala/io/janstenpickle/trace4cats/avro/AvroSpanExporter.scala

```diff
@@ -98,22 +98,23 @@ object AvroSpanExporter {
           .start
       )(fiber =>
         Applicative[F].unit.whileM_(for {
-          queueNonEmpty <- queue.getSize.map(_ != 0)
+          queueNonEmpty <- Sync[F].delay[Boolean](
```

Comment:
`cats.effect.std.Queue` doesn't support peeking for now. Something needs to be done here...